.gitignore

```diff
@@ -207,3 +207,4 @@ p2p.log
 
 ## Strings.java is being generated
 Strings.java
+!/src/ch/imedias/rsccfx/view/RsccRequestView.java
```

Comment:
I don't know... for whatever reason it showed to me I can't push the file since it's ignored.. that's why I added it....